frontend/src/app/schema/accounts.js

```diff
@@ -123,6 +123,14 @@ export default {
                     }
                 }
             },
+            undeletable: {
+                type: 'string',
+                enum: [
+                    'OWNER',
+                    'OWN_BUCKET',
+                    'NONE'
```

Comment:
Why do we need `NONE` at any other place when an item is deletable we just don't send the property (see bucket and resource). Please line this with the other implementations.